user/tests/wiring/no_fixture_long_running/network.cpp

```diff
@@ -207,13 +207,18 @@ test(NETWORK_02_network_connection_recovers_after_ncp_failure) {
 
     // Simulate NCP failure by reconfiguring the NCP serial port with a baudrate
     // setting different from expected.
-    // FIXME: when a new platform is added not using HAL_USART_SERIAL2 or not using
+    // FIXME: when a new platform is added not using HAL_PLATFORM_CELLULAR_SERIAL or not using
     // UART for talking to NCP.
     SINGLE_THREADED_BLOCK() {
 #if HAL_PLATFORM_NCP_AT
-        hal_usart_end(HAL_USART_SERIAL2);
-        hal_usart_begin_config(HAL_USART_SERIAL2, 57600, SERIAL_8N1, nullptr);
-#endif
+#if HAL_PLATFORM_CELLULAR
```

Comment:
This will change later?

Reply:
I dont believe so. `HAL_PLATFORM_CELLULAR` is intended to pair with `HAL_PLATFORM_CELLULAR_SERIAL` below. IE if platform cellular is defined, then there should be a serial interface for that cellular defined as well.